xds/src/main/java/io/grpc/xds/internal/rbac/engine/GrpcAuthorizationEngine.java

```diff
@@ -316,9 +335,39 @@ private String getPath() {
 
     @Nullable
     private String getHeader(String headerName) {
-      if (headerName.endsWith(Metadata.BINARY_HEADER_SUFFIX)) {
+      if ("TE".equals(headerName)) {
         return null;
       }
+      if ("Host".equals(headerName)) {
+        headerName = "authority";
+      }
+      String value = deserializeHeader(headerName);
+      if (value == null && ":method".equals(headerName)) {
```

Comment:
There's no code for `:path` here. If you are aware and were going to do it later, that's fine.